source/compliance/nist/malware-detection.rst

```diff
@@ -11,7 +11,6 @@ Wazuh offers several capabilities that support malware detection. The Malware de
 - File integrity monitoring and threat detection rules.
 - Rootkit behavior detection rules.
 - CDB lists and threat intelligence to detect and remove malicious files
-- VirusTotal integration for malware detection.
```

Comment:
Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."